src/common/Custom.tsx

```diff
@@ -0,0 +1,15 @@
+import { User } from "./interface";
+import chatData from "../json/chatRoom.json";
+import userData from "../json/userData.json";
+
+const chatrooms = chatData.rooms;
+const users = userData.users;
+
+export const getRoomMembers = (roomId: number, includeCurUser: Boolean) => {
+  const roomMembers: User[] = [];
+  chatrooms[roomId].users.map((userId) => roomMembers.push(users[userId]));
+  if (!includeCurUser) {
+    roomMembers.shift();
+  }
+  return roomMembers;
+};
```

Comment:
리액트에서는 객체와 배열을 업데이트할 때 불변성을 지키는 것이 중요합니다. 
즉, 객체나 배열을 직접 수정하지 않고 값을 변경하고 싶은 경우 새로운 객체나 배열을 생성하는 것을 말합니다. 이는 리액트가 상태의 변화를 알아차리고 렌더링을 일으키는 방식의 특징 때문인데 이는 나중에 한 번 찾아보시면 좋을 것 같아요! 
따라서 기존의 경우에 따라 기존의 배열을 shift해서 변경하는 것 보다는 각각의 조건에 맞는 새로운 배열을 만들어 리턴하는 것이 어떨까 싶습니다. 
```suggestion
export const getRoomMembers = (roomId: number, includeCurUser: Boolean) => {
  
  if(includeCurUser){
    const roomMembers: User[] = [];
    chatrooms[roomId].users.map((userId) => roomMembers.push(users[userId]))
    return roomMembers;

  }else{
    const allMembers: User[] = [];
    chatrooms[roomId].users.map((userId) => {if(userId!==0){allMembers.push(users[userId])}});
    return allMembers;
  }
};
```